public/request/leaderboard/remove-entry.php

```diff
@@ -13,8 +16,14 @@
 
 if (removeLeaderboardEntry($targetUser, $leaderboardId)) {
     header('Location: ' . $returnUrl . '&success=true');
-    $commentText = 'removed "' . $targetUser . '"s entry from this leaderboard';
-    addArticleComment("Server", \RA\ArticleType::Leaderboard, $leaderboardId, "\"$user\" $commentText.", $user);
+    if ($targetUser != $user) {
+        if ($reason === '') {
```

Comment:
`empty($reason)` is more robust for when the `r` query parameter is not set at all. Unlikely to happen, yet the condition's result would be unexpected if the form is submitted without it down the line for whatever reason.